octavia-cli/unit_tests/test_list/test_connectors_definitions.py

```diff
@@ -0,0 +1,151 @@
+        definitions = ConnectorsDefinitions(*connectors_definitions_mock_args)
+        api_response = {definitions.response_definition_list_field: []}
+        for i in range(5):
+            definition = {field: f"{field}_value_{i}" for field in definitions.fields_to_display}
+            definition["discarded_field"] = "discarded_value"
+            api_response[definitions.response_definition_list_field].append(definition)
+        parsed_definitions = definitions._parse_response(api_response)
+        assert len(parsed_definitions) == 5
+        for i in range(5):
+            assert parsed_definitions[i] == [f"{field}_value_{i}" for field in definitions.fields_to_display]
+            assert "discarded_value" not in parsed_definitions[i]
+
+    def test_latest_definitions(self, patch_base_class, mocker, connectors_definitions_mock_args):
+        mock_list_latest_definitions = connectors_definitions_mock_args[-1]
+        mocker.patch.object(ConnectorsDefinitions, "_parse_response")
+        definitions = ConnectorsDefinitions(*connectors_definitions_mock_args)
+        assert definitions.latest_definitions == definitions._parse_response.return_value
+        mock_list_latest_definitions.assert_called_with(definitions.api_instance, **definitions.LIST_LATEST_DEFINITIONS_KWARGS)
+        definitions._parse_response.assert_called_with(mock_list_latest_definitions.return_value)
+
+    def test_repr(self, patch_base_class, mocker, connectors_definitions_mock_args):
+        headers = ["fieldA", "fieldB", "fieldC"]
+        latest_definitions = [["a", "b", "c"]]
+        mocker.patch.object(ConnectorsDefinitions, "fields_to_display", headers)
+        mocker.patch.object(ConnectorsDefinitions, "latest_definitions", latest_definitions)
+        mocker.patch.object(ConnectorsDefinitions, "_display_as_table")
+        mocker.patch.object(ConnectorsDefinitions, "_format_column_names")
+
+        definitions = ConnectorsDefinitions(*connectors_definitions_mock_args)
+        representation = definitions.__repr__()
+        definitions._display_as_table.assert_called_with([definitions._format_column_names.return_value] + latest_definitions)
+        assert representation == definitions._display_as_table.return_value
+
+    @pytest.mark.parametrize(
+        "test_data,padding,expected_col_width",
+        [([["a", "___10chars"], ["e", "f"]], 2, 2 + 10), ([["a", "___10chars"], ["e", "____11chars"]], 2, 2 + 11), ([[""]], 2, 2)],
+    )
+    def test_compute_col_width(self, test_data, padding, expected_col_width):
+        col_width = ConnectorsDefinitions._compute_col_width(test_data, padding)
+        assert col_width == expected_col_width
+
+    @pytest.mark.parametrize(
+        "test_data,col_width,expected_output",
+        [
+            ([["a", "___10chars"], ["e", "____11chars"]], 13, "a            ___10chars   \ne            ____11chars  "),
+        ],
+    )
+    def test_display_as_table(self, mocker, test_data, col_width, expected_output):
```

Comment:
As noted above, these tests should also be moved to a formatting module when you move the underlying methods